src/main/java/edu/cornell/kfs/sys/dataaccess/impl/DocumentMaintenanceDaoJdbc.java

```diff
@@ -37,23 +37,27 @@
     }
 
     private CuSqlQuery buildRequeueSqlQuery() {
-        CuSqlChunk sqlChunk = buildRequeueSqlQueryChunk();
-        sqlChunk.append(" ORDER BY DOC_HDR_ID ASC");
+        CuSqlChunk sqlChunk = buildRequeueSqlQueryChunk(true);

```

Comment:
I understand now why you wanted to avoid using booleans in the parameter lists.

Do you think it would be clearer as to what the boolean values mean when we look at this code in a year from now if we had the following two method wrapper names for the boolean use as
- buildRequeueSqlQueryWithOrderBy() {buildRequeueSqlQueryChunk(true)}
- buildRequeueSqlQueryWithoutOrderBy() {buildRequeueSqlQueryChunk(false)}

where this private method on line 39 is renamed to buildRequeueSqlQueryWithOrderBy()
and the method call on line 83 is changed to buildRequeueSqlQueryWithoutOrderBy()?

Reply:
Yes, I agree that adding wrapper methods like those will help make the code's intent much clearer. I'll adjust this code accordingly.